### ISSUE TITLE
有没有办法把tags语法的"-" 改成#,并且多个tags不用换行

### ISSUE THREAD

**Issue by digitalstationol** [devbean/obsidian-wordpress#42]
有没有办法把tags语法的"-" 改成#,并且多个tags不用换行

**Comment by devbean** [comment]
您的意思是使用这样的语法：
```
---
tags: #tag1 #tag2 #tag3
---
```

**Comment by digitalstationol** [comment]
> 您的意思是使用这样的语法：
> 
> ```
> ---
> tags: #tag1 #tag2 #tag3
> ---
> ```

是的,大神,这样的好处是可以和obsidian本身的tag融合,多标签的情况下又不会占用太多行,一举多得,如果技术上可以实现,希望大佬可以至少加一个开关,谢谢了!

**Comment by devbean** [comment]
好像 Obsidian 也是支持 YAML 中的 tags 的语法：[https://help.obsidian.md/Editing+and+formatting/Tags](https://help.obsidian.md/Editing+and+formatting/Tags)，不确定是不是也是可以的？

**Comment by digitalstationol** [comment]
> tags: #tag1 #tag2 #tag3

我觉得应该是可以的 因为 yaml中的tags: #tag1 #tag2 #tag3,在我切换成为阅读模式的时候,#tag1 #tag2 #tag都变成了紫色(默认主题),所有如果插件支持这种形式的tag,那就是两全其美了,谢谢你的插件!
---
tags: #tag1 #tag2 #tag3
---

**Comment by devbean** [comment]
我看了一下，Obsidian 是可以识别
```
tags:
  - tag1
  - tag2
```
这样的语法的；会把数组每一个元素当做一个 tag，可以检索。写成
```
tags: #tag1 #tag2
```
反而不可以。
按照文档，写成
```
tags: tag1, tag2
```
也是可以的。

另外，我试着切换到阅读模式，YAML 里面的内容都会隐藏，并不会显示出来。

还有一个插件，FuzzyTag，可以在 YAML 中自动完成 tag，感觉结合使用更好。
